justice/mock_data.py

```diff
@@ -26,3 +26,23 @@ def noisify_obs(y, scatter):
     errs = scatter * np.ones_like(y)
     new_y = y + sps.norm(0., scatter).rvs(np.shape(y))
     return(new_y, errs)
+
+LC = namedtuple('LC', ('x', 'y'))#, 'yerr'))
+
+def transform(lc, deltax, deltay, stretchx, stretchy):
+    new_x = (stretchx * lc.x) + deltax
+    new_y = (stretchy * lc.y) + deltay
+    # new_yerr = np.sqrt(np.abs(new_y / lc.y) * lc.yerr**2)
+    return LC(new_x, new_y)#, lc.yerr)
+
+def make_dataset(num_obj, def_cadence, cls_models, cls_params, cls_wts=None):
+    num_cls = len(cls_models)
+    lcs = []
+    truth = np.random.choice(range(num_cls), num_obj, p=cls_wts)
+    ids, inds, cts = np.unique(truth, return_counts=True, return_inverse=True)
```

Comment:
why do you need this? seems like you could do

```
for cls_id in truth:
    # same loop body, just use `cls_id` in place of ids[inds[i]]
```

also not sure about the variable naming "truth"?

Reply:
Excellent points!  I've addressed these issues and checked that the test still runs.